### ISSUE TITLE
yahoo changed login screen

### ISSUE THREAD

**Issue by cdenny** [isaacmiller2004/GetYahooFBBData#1]
Yahoo updated the login screen to have two screens in login.
Now the username and password are entered in seperate pages.

**Comment by jrcapriles** [comment]
Hey @cdenny, I had the same problem but I found a quick fix, you just need to modify the "Authentication and Cookie Handling" section. This is how my login block's looks:

```
    br.select_form(nr=0)
    br.form["username"] = username
    submit_response = br.submit()
    br.select_form(nr=0)
    br.form["passwd"] = password
    br.submit()
```

**Comment by cdenny** [comment]
@jrcapriles yup, I figured that out a couple weeks ago.  Python is not my first lang.    I added a fork with the updated and works fine.   :)   just glad I found this code :)

**Comment by isaacmiller2004** [comment]
@cdenny Thanks for letting me know. It's updated now. And I'm glad you found this useful!
